docs/docs/connectors/your-own-website.mdx

```diff
@@ -10,7 +10,7 @@ is usually the chat interface that ships with Rasa X, where you can [invite user
 to test your bot](https://rasa.com/docs/rasa-x/user-guide/share-assistant/#share-your-bot).
 
 If you already have an existing website and want to add a Rasa assistant to it,
-you can use [Chatroom](https://github.com/scalableminds/chatroom), a widget which you can incorporate into your existing webpage by adding a HTML snippet.
+you can use [Rasa Webchat](#webchat-widget), a widget which you can incorporate into your existing webpage by adding an HTML snippet.
```

Comment:
```suggestion
you can use [Rasa Webchat](#webchat-widget), a widget which you can incorporate into your existing webpage by adding an HTML snippet.
```
Did we confirm with marketing on the name `Rasa Webchat`?

Reply:
didn't kick of the project so not sure. let me check in with them - good point.